docs/sources/staged.rst

```diff
@@ -2,7 +2,7 @@ Source: staged
 ==============
 
 The ``staged`` push source allows the loading of content from any locally accessible
-directory referred to as the staging area. Content types and destinations are defined
+directory referred to as the staging area. Content types and destinations s and destinations are defined
```

Comment:
Looks like you added some junk into your commit, can you please clean it up?

I'm not sure what your workflow is, but if you are currently doing something like `git commit -a` you might want to give `git add -p` a try which allows you to review every chunk you're adding into your commits.

The change looks good apart from this.

Reply:
I have a "quick amend" git alias, which is usually fine, but I think I must have fat fingered that docs file while researching  kickstarts.  The `-p` arg sounds handy though :+1: